controllers/powerprofile_controller_test.go

```diff
@@ -451,12 +460,16 @@ func TestPowerProfile_Reconcile_MaxMinValuesZero(t *testing.T) {
 		testCase    string
 		nodeName    string
 		profileName string
+		validateErr func(e error) bool
 		clientObjs  []runtime.Object
 	}{
 		{
 			testCase:    "Test Case 1 - Max value zero",
 			nodeName:    "TestNode",
 			profileName: "user-created",
+			validateErr: func(e error) bool {
+				return assert.ErrorContains(t, e, "max frequency value cannot be lower than the minimum frequency value")
```

Comment:
Please be consistent when abbreviating.

```suggestion
				return assert.ErrorContains(t, e, "max frequency value cannot be lower than the min frequency value")
```

Reply:
Will address all changes to controllers/controller tests in the upcoming release